### PR TITLE
docs: add crd page for consistency

### LINKED ISSUE #400: Make CRD page appear consistent with rest of doc pages
### Who is this for and what problem do they have today?
See:
https://doc.crds.dev/github.com/vectorizedio/redpanda

The Operator Custom Resource Definition (CRD) page takes readers outside the context of RP product docs:
- it does not look like any doc page
- there is no description or guidance for customers with regard to what they do on this page
- this page does not provide the customer experience RP docs strive to achieve

### What would you like fixed in or added to the documentation?

This page must be repurposed/re-presented as RP documentation, which means there should be:
- an H1 heading "Redpanda Operator custom resource definition (CRD)"
- Left sidenav to rest of docs should not be removed. In other words, customers should not be taken out of doc context to read this page. One alternative: create a doc page describing what CRD is, the value, the benefit, the context in which you would access/use, etc. This page would link to the page now displayed as a "doc page" (it's just not).
- an introductory paragraph describing:
    - What the CRD is
    - Why and when it should be used
    - guidance on what to expect as you click through the links (at each juncture when I clicked through the links, none of the pages provide descriptions or guidance)

### Is this a new feature or a gap in the documentation?

This is neither a new feature nor gap--this is a usability and consistency issue that must be addressed.

#### If new feature, what's the expected release number and date? 
n/a

### Additional notes
@Deflaimun I seem to remember that there was an issue for this page way back during my first week at RP. Searched for the issue and could not find so that I could see/understand how we are left with the current page. If you know what issue that was, please share here. Thanks!

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves #400 
https://deploy-preview-824--redpanda-documentation.netlify.app/docs/platform/reference/redpanda-operator/crd/

### CODE REVIEW

File: docs/platform/reference/redpanda-operator/crd.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Operator Custom Resource Definition (CRD)
+---
+
+<head>
+    <meta name="title" content="Operator Custom Resource Definition (CRD)| Redpanda Docs"/>
+    <meta name="description" content="Operator Custom Resource Definition (CRD)."/>
+</head>
+
+Redpanda offers a Operator Custom Resource Definition (CRD).
```
Comment (micheleRP) [inline docs/platform/reference/redpanda-operator/crd.mdx:10]:
```suggestion
Redpanda offers a CRD for the Redpanda Operator. 
```

File: docs/platform/reference/redpanda-operator/crd.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+---
+
+<head>
+    <meta name="title" content="Operator Custom Resource Definition (CRD)| Redpanda Docs"/>
+    <meta name="description" content="Operator Custom Resource Definition (CRD)."/>
+</head>
+
+Redpanda offers a Operator Custom Resource Definition (CRD). 
+
+You can check the file at [Operator Custom Resource Definition reference page](https://doc.crds.dev/github.com/vectorizedio/redpanda).
```
Comment (micheleRP) [inline docs/platform/reference/redpanda-operator/crd.mdx:12]:
```suggestion
See the [Redpanda Operator Custom Resource Definition](https://doc.crds.dev/github.com/vectorizedio/redpanda).
```

**Review by micheleRP — APPROVED** [review]
Hi @Paulo. A couple small suggestions for the new page, then approved.

**Comment by Deflaimun** [comment]
Thank you for the edits @micheleRP

**Comment by Deflaimun** [comment]
@paulo HOW DID YOU GOT PAULO ON GITHUB? hahaaha Thank you for being a great sport about it.

**Pull request merged**